### PR TITLE
updated codelists

### PR DESCRIPTION
**Pull request by JimBryant7**
Added most up to date code lists

Just requires a sanity check by anyone.

### CODE REVIEW

File: geo-specific-codelist/gremlin/output-area.grm.zip
Comment (sandypadmanabhan) [inline geo-specific-codelist/gremlin/output-area.grm.zip:1]: This is a zip file , will that be ok to upload?

Comment (JimBryant7) [reply geo-specific-codelist/gremlin/output-area.grm.zip:1]: zip file is not okay to upload however the full file is over 100MB (github gave me a exceeds file size error message) so I zipped it so it is still available if needed.
I could split the .grm file into 2 smaller files if you think that is a better solution but I worry that someone may not realise there are 2 files if they needed to upload

Comment (sandypadmanabhan) [reply geo-specific-codelist/gremlin/output-area.grm.zip:1]: 2 files for upload would be good

Comment (JimBryant7) [reply geo-specific-codelist/gremlin/output-area.grm.zip:1]: I have removed the code list from the repo as it is not used anywhere

File: code-list-scripts/gremlin/census_2021_year_arrival_uk (1).grm [hunk 1]
```diff
@@ -0,0 +1,112 @@
+g.V().hasId('_code_list_census_2021_year_arrival_uk_one-off').
```
Comment (sandypadmanabhan) [inline code-list-scripts/gremlin/census_2021_year_arrival_uk (1).grm:1]: everycode list id has `hypen` but here is underscore and the title has (1) in it.  Not sure if this is intended

Comment (JimBryant7) [reply code-list-scripts/gremlin/census_2021_year_arrival_uk (1).grm]: good spot, ive amended the file name

Comment (sandypadmanabhan) [reply code-list-scripts/gremlin/census_2021_year_arrival_uk (1).grm]: The list id property has  'underscore' instead of 'hypen' is that ok?

**Review by sandypadmanabhan — APPROVED** [review]
LGTM

**Pull request merged**